src/pages/Home.module.css

```diff
@@ -1,14 +1,58 @@
-.hero{
-    height:17rem;
+.hero {
+  height: 17rem;
 }
+
 .btn {
-    border-radius:var(--border-radius1);
-    color: var(--secondry-color);
-    font-size: var(--size-1);
-    background: var(--red-colorl1);
-    padding: 10px 20px 10px 20px;
-    text-decoration: none;
-  }
-  .btn:hover {
-    background: var(--red-colord2);
-  }
\ No newline at end of file
+  border-radius: var(--border-radius1);
+  color: var(--secondry-color);
+  font-size: var(--size-1);
+  background: var(--red-colorl1);
+  padding: 10px 20px 10px 20px;
```

Comment:
you can use shorthand to implement vertical and horizontal padding accordingly
`padding: 10px 20px`
also do use rems for px greater than 10-16